src/aspire/source/micrograph.py

```diff
@@ -1,75 +1,389 @@
 import logging
+import os
+from abc import ABC, abstractmethod
+from glob import glob
+from pathlib import Path
 
 import numpy as np
 
 from aspire.image import Image
 from aspire.source import Simulation
 from aspire.source.image import _ImageAccessor
+from aspire.storage import StarFile
 from aspire.utils import Random, grid_2d
+from aspire.volume import Volume
 
 logger = logging.getLogger(__name__)
 
 
-class MicrographSimulation:
+class MicrographSource(ABC):
```

Comment:
No. Micrographs don't have a symmetry.  A symmetry of the associated volume could be written to the star files for the sim case, but i beleive all the metadata for CoordinateSource is broken anyway, soooo it would not actually do anything. We also don't actually have any symmetry based optimizations today, so let's not make more make work.